packages/teams-js/src/internal/messageObjects.ts

```diff
@@ -1,9 +1,57 @@
+import { generateGUID, validateUuid } from './utils';
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ *
+ * MessageUUID object that holds the uuid used for processing MessageRequests and MessageResponses
+ */
+export class MessageUUID {
```

Comment:
>MessageUUID [](http://example.com/codeflow?start=13&length=11)

This seems like a valuable and useful general purpose class that the library should use for holding UUIDs across the codebase. Would you be willing to rename it to something less-specific to messages and put it in a general shared location?